### ISSUE TITLE
Carthage docs & example

### ISSUE THREAD

**Issue by bootstraponline** [google/EarlGrey#72]
Now that Carthage is able to fetch and build EarlGrey, it'd be nice to have an example similar to the existing cocoapods demo. Also the `install-and-run` doc should be updated to include the steps necessary to get started with carthage.

**Comment by khandpur** [comment]
Yay! would love to but I have very limited experience with Carthage.

**Comment by bootstraponline** [comment]
I'm waiting on permission from my employer but I hope to have something to contribute in this area soon.

**Comment by tirodkar** [comment]
Hey @bootstraponline, is this still being worked on?

**Comment by bootstraponline** [comment]
Yep. First step was the gem, next is the docs.

**Comment by bootstraponline** [comment]
https://github.com/google/EarlGrey/pull/94 adds a cocoapods test, after that is updating the docs.
